content/docs/5.1/deploy/install/test.md

```diff
@@ -10,27 +10,27 @@ This guide describes how to install and deploy TiKV for test using TiUP playgrou
 
 ## TiUP Playground
 
-This chapter describes how to deploy a TiKV cluster using TiUP Playground.
+This chapter describes how to deploy a TiKV cluster using TiUP playground.
```

Comment:
TiUP Playground is a component name.

Reply:
But in other places of this file, it is written as "playground". Is it necessary to unify the style?

Reply:
If "playground" is used in similar places, we can keep it.